### DISCUSSION TITLE
RX6650xt slow to load

### DISCUSSION THREAD

**Discussion by blackjw1212** [ChefKissInc/NootRX#31] [Q&A] [answered]
After upgrading RX6650XT from RX570 in Monterey 12.7.3, the opencore execution speed is very slow.
[Uploading EFI.zip…](https://drive.google.com/file/d/1OCtyrHngZqsrbbr9haqMCQo3xwEDvUfD/view?usp=sharing)

**Comment by VisualEhrmanntraut** [comment, marked as answer]
Download a release build, not debug.

**Reply by blackjw1212** [reply]
Need to close bios igfx 🥹
